### PR TITLE
Add default dhcp_relay.yml file to OneImage build

### PR DESCRIPTION
**Pull request by jleveque**
Fixes https://github.com/Azure/sonic-buildimage/issues/330.

### CODE REVIEW

File: files/build_templates/sonic_debian_extension.j2 [hunk 1]
```diff
@@ -111,6 +111,9 @@ sudo bash -c "echo enabled=false > $FILESYSTEM_ROOT/etc/sonic/updategraph.conf"
 # Copy SNMP configuration files
 sudo cp $IMAGE_CONFIGS/snmp/snmp.yml $FILESYSTEM_ROOT/etc/sonic/
 
+# Copy initial DHCP relay configuration file, will be overwritten after docker starts
```
Comment (taoyl-ms) [inline files/build_templates/sonic_debian_extension.j2]: No I don't think anyone will overwrite this file. This file is the input of config.sh for dhcp relay docker and the real configuration will be generated according to this file.

File: files/image_config/dhcp_relay/dhcp_relay.yml [hunk 2]
```diff
@@ -0,0 +1 @@
+dhcp_servers: []
```
Comment (taoyl-ms) [inline files/image_config/dhcp_relay/dhcp_relay.yml]: Just to double confirm - have you verified that dhcp relay docker can start correctly with this empty value?

File: files/build_templates/sonic_debian_extension.j2 (hunk 1 above)
Comment (jleveque) [inline files/build_templates/sonic_debian_extension.j2]: So until the DHCP servers are stored in the minigraph, this file must be edited to contain the actual DHCP server IPs?

**Comment by taoyl-ms** [comment]
Just discussed offline with Joe - we have agreed that we should embed this information into minigraph. I'll submit another PR for it.

**Comment by jleveque** [comment]
This PR will be abandoned in favor of a solution that pulls the DHCP servers from the minigraph. I will update this comment and close this PR once I submit a new PR.

**Comment by jleveque** [comment]
Abandoning this PR in favor of https://github.com/Azure/sonic-buildimage/pull/374.

**Pull request closed without merging**